library/vtproto/v0.3.0/buf.plugin.yaml

```diff
@@ -0,0 +1,13 @@
+version: v1
+name: buf.build/library/vtproto
+plugin_version: v0.3.0
+output_languages: [go]
+source_url: https://github.com/planetscale/vtprotobuf
+description: Generates optimized marshall & unmarshal code.
+default_opts:
+  - paths=source_relative
+registry:
+  go:
+    deps:
+      - module: google.golang.org/protobuf
+        version: v1.28.1
```

Comment:
I presume the latest version will work, although I do wonder if we want to match this closer to the go.mod file?

https://github.com/planetscale/vtprotobuf/blob/main/go.mod#L8

cc @pkwarren thoughts?

Reply:
I think we should remove the registry section - it currently generates code which has to live in the same package as the generated protos. With the default features, it also requires grpc.

People using this will need to use `buf.gen.yaml` until we get a chance to patch plugins to work together.

Reply:
Yep, let's remove registry config, esp. for plugins that output to the same package along with the base types.